Mathlib/CategoryTheory/EffectiveEpi/Enough.lean

```diff
@@ -62,9 +62,9 @@ instance (X : D) : EffectiveEpi (F.effectiveEpiOver X) :=
 /-- An effective presentation of an object with respect to an equivalence of categories. -/
 def equivalenceEffectivePresentation (e : C ≌ D) (X : D) :
     EffectivePresentation e.functor X where
```

Comment:
If you want, you could also put the entire `def` on one line (it's 99 chars overall)